airbyte-webapp/src/components/ui/Flex/FlexContainer.tsx

```diff
@@ -27,46 +27,54 @@ interface FlexContainerProps {
  * Renders a div element which layouts its children as flex items as specified by the props.
  *  Children of a `FlexContainer` can but don't have to be `FlexItem` elements.
  */
-export const FlexContainer: React.FC<React.PropsWithChildren<FlexContainerProps & HTMLAttributes<HTMLDivElement>>> = ({
-  className,
-  direction = "row",
-  gap = "md",
-  alignItems = "stretch",
-  justifyContent = "flex-start",
-  children,
-  ...otherProps
-}) => {
-  const fullClassName = classNames(
+export const FlexContainer = React.forwardRef<
```

Comment:
Had to make this FlexContainer a forwardRef, so that the useResizeDetector hook ref could be set on it